packages/mux-uploader/README.md

```diff
@@ -61,10 +61,7 @@ If you are using ECMAScript modules, you can also load the `mux-uploader.mjs` fi
 <body>
   <p></p>
 
-  <mux-uploader
-    url="authenticated-url"
-    id="uploader"
-  ></mux-player>
+  <mux-uploader url="authenticated-url" id="uploader"></mux-uploader>
 </body>
```

Comment:
can we expand on this example, or add another example about how to use `<mux-uploader-drop>`? Based on these docs I'm not sure how to use that other component